### PR TITLE
Maya: Update Screenshots

### PR DESCRIPTION
**Pull request by MustafaJafar**
# Changelog Description

Update screenshots on these pages as well as using a better location to save these images.
- [x] `docs/addon_maya_artist` 
- [x] `docs/addon_maya_arnold_artist` 
- [x] `docs/addon_maya_admin`

For consistency, I used updated the location of images on these pages:
- [x] `docs/addon_maya_redshift_artist`
- [x] `docs/addon_maya_vray_artist`
- [x] `docs/addon_maya_multiverse_artist`
- [x] `docs/addon_maya_yeti_artist`

### CODE REVIEW

File: website/docs/addon_maya_artist.md
Comment (tokejepsen) [inline website/docs/addon_maya_artist.md:42]: if we have a subfolder of `maya`, should we remove `maya` from the file name?

Comment (MustafaJafar) [reply website/docs/addon_maya_artist.md:42]: That's fair!

File: website/docs/addon_maya_arnold_artist.md
Comment (tokejepsen) [inline website/docs/addon_maya_arnold_artist.md:24]: I think splitting the images into these workflow folder might be too much. Having `admin` and `artist` would be enough and it pretty clear where images should go.

**Review by tokejepsen — CHANGES_REQUESTED** [review]

**Review by tokejepsen — APPROVED** [review]
Looks good to me.

**Comment by MustafaJafar** [comment]
Do we need a second opinion before clicking merge pull request ?

**Comment by tokejepsen** [comment]
> Do we need a second opinion before clicking merge pull request ?

Yeah, need another approval before we'll merge. Second approver usually merges.

**Pull request merged**
